modules/lvms-provisioning-storage-using-logical-volume-manager-operator.adoc

```diff
@@ -35,12 +41,16 @@ spec:
   resources:
     requests:
       storage: 10Gi <3>
-  storageClassName: lvms-vg1 <4>
+    limits:
+      storage: 20Gi <4>
+  storageClassName: lvms-vg1 <5>
 ----
 <1> Specify a name for the PVC.
 <2> To create a block PVC, set this field to `Block`. To create a file PVC, set this field to `Filesystem`.
-<3> Specify the storage size. The total storage size that you can provision is limited by the size of the Logical Volume Manager (LVM) thin pool and the overprovisioning factor.
-<4> The value of the `storageClassName` field must be in the format `lvms-<device_class_name>`, where, `<device_class_name>` is the value of the `deviceClasses.name` field in the `LVMCluster` CR.
+<3> Specify the storage size. By default, this field is set to 1 GiB. If the value of this field is less than the minimum storage size, the requested storage size is
```

Comment:
Storage size field in PVC can not be empty, so I think there is no default value of '1GiB' is set for this field.

Reply:
Hi @radeore 
Sure, updated as per the suggestion.

Reply:
@jakobmoellerdev 
Could you PTAL at the above suggestion?
Thanks!

Reply:
I think this is (partially) correct. It is true that this field is not set to 1GiB but not specifying a request will cause the Driver to allocate 1 GiB by default. Thats why imo, we should slightly change the wording but keep a comment

Reply:
@jakobmoellerdev 
Is it possible to not specify this field (meaning excluding this from the configuration)? Because @radeore mentions that this field cannot be empty!

If "requests.storage" is a Mandatory field, then does that not imply that it can neither be empty nor be excluded from the configuration?

Not sure if I am missing something here!

Reply:
Thanks for the correction. I double checked this and it seems that K8s is enforcing a limit here even though the underlying CSI driver has a defaulting logic. Thus it is safe to assume the mandatory field as you suggested @radeore

Reply:
@radeore @jakobmoellerdev 
Thanks for the confirmation!